br/pkg/lightning/mydump/parquet_parser.go

```diff
@@ -566,3 +576,14 @@ func (*ParquetParser) SetColumns(_ []string) {
 func (pp *ParquetParser) SetLogger(l log.Logger) {
 	pp.logger = l
 }
+
+func jdToTime(jd int32, nsec int64) time.Time {
+	sec := int64(jd-jan011970) * secPerDay
+	return time.Unix(sec, int64(nsec))
+}
+
+func int96ToTime(parquetDate []byte) time.Time {
```

Comment:
can you refer to a parquet encoding for timestamp as comment?

Reply:
https://github.com/apache/spark/blob/d66a4e82eceb89a274edeb22c2fb4384bed5078b/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetWriteSupport.scala#L171-L178

maybe this is enough

Reply:
2536ce5